statuspage.json

```diff
@@ -0,0 +1,1712 @@
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table",
+                "width": 12,
+                "height": 5,
+                "title": "create incident"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "update incident",
+            "description": "Update an incident",
+            "type": "generic",
+            "identifier": "update_incident",
+            "read_only": false,
+            "parameters": {
+                "incident_id": {
+                    "description": "Incident Identifier",
+                    "data_type": "string",
+                    "order": 0,
+                    "contains": [
+                        "incident id"
+                    ],
+                    "required": true,
+                    "primary": true
+                },
+                "name": {
+                    "description": "Update Incident Name",
+                    "data_type": "string",
+                    "order": 1,
+                    "contains": [
+                        "incident name"
+                    ],
+                    "primary": true
+                },
+                "status": {
+                    "description": "Update Incident status",
+                    "data_type": "string",
+                    "order": 2,
+                    "contains": [
+                        "incident status"
+                    ],
+                    "value_list": [
+                        "investigating",
+                        "identified",
+                        "monitoring",
+                        "resolved"
+                    ],
+                    "primary": true
+                },
+                "impact_override": {
+                    "description": "Select value to override impact value",
+                    "data_type": "string",
+                    "order": 3,
+                    "contains": [
+                        "incident impact value"
+                    ],
+                    "value_list": [
+                        "none",
+                        "maintenance",
+                        "minor",
+                        "major",
+                        "critical"
+                    ],
+                    "primary": true
+                },
+                "body": {
+                    "description": "Update initial message, created as the first incident update",
+                    "data_type": "string",
+                    "order": 4,
+                    "contains": [
+                        "incident body"
+                    ],
+                    "primary": true
+                },
+                "component_ids": {
+                    "description": "Specify component's id affected by this incident. Separate multiple ids with commas",
+                    "data_type": "string",
+                    "order": 5,
+                    "example": [
```

Comment:
Remove "example" key from parameter. It should be added as "example_values" in output list.

Reply:
Removed **example** key from parameter.